lib/ork/model/document.rb

```diff
@@ -69,13 +69,20 @@ def update(attributes)
     #   u = User.new(:name => "John").save
     #   # => #<User:6kS5VHNbaed9h7gFLnVg5lmO4U7 {:name=>"John"}>
     #
-    def save
+    def save(options = {})
+      quorum_types = [:r, :w, :dw]
+      quorum = options.fetch(:quorum, {})
+
+      unless quorum.empty?
+        raise ArgumentError.new("invalid quorum option") unless quorum.keys.select { |key| !quorum_types.include?(key) }.empty?
```

Comment:
A good practice is to create a specific error like `InvalidQuorumOption < ArgumentError`
Also, if you have to map/select a collection, please use a different variable.

BTW did you consider to create a Quorum class to handle all the methods around quorum?